internal/scalibrplugin/presets.go

```diff
@@ -148,16 +148,19 @@ var enricherPresets = map[string]enricherlist.InitMap{
 }
 
 func baseImageEnricher() enricher.Enricher {
+	// The grpc client **does not** make any requests are start in an IDLE state until
```

Comment:
umm this reads weirdly - it looks like you couldn't decide between "the client does not make any request until ..." and "the client starts in an IDLE state until ...", and so mashed them together 😅